kivy/graphics/transformation.pyx

```diff
@@ -611,6 +611,20 @@ cdef class Matrix:
 
         return (winx, winy, winz)
 
+    def transform(self, list point):
```

Comment:
`double` seems to be the best way to go @matham. Some of Kivent's points were stored as lists so that was the reason for this, but there could be use cases for tuples.